reference/CloudPay-API-Specification.yaml

```diff
@@ -1381,6 +1381,143 @@ paths:
 
         **Note** *Only one of the four authorisation parameters are required*
     parameters: []
+  /api/v1/account/transactions:
+    get:
+      operationId: get-api-v1-account-transactions
+      summary: Retrieve the authenticated customer's transactions
+      tags:
```

Comment:
Same issue as mentioned in previous PRs about multiple tags...

Reply:
i can leave only 
tags:
        - Account
right?

Reply:
yes please, one tag that describes it best 👍